src/main/scala/org/deepdive/extraction/ExtractorRunner.scala

```diff
@@ -400,7 +398,7 @@ class ExtractorRunner(dataStore: JsonExtractionDataStore, dbSettings: DbSettings
 
     // NEW: for mysqlimport compatibility, the file basename must be same as table name.
     val queryOutputFile = new File(queryOutputPath + s"${outputRel}.copy_query_${funcName}.tsv")
-    val gpFileName = s"${outputRel}_unload_${funcName}"
+    val gpFileName = s"${outputRel}_query_unload"
```

Comment:
I just further updated the naming with `${funcName}_query_unload`, since it is unique and could prevent potential parallelism problems. I also added comments to this change. Waiting for the test.